src/develop/develop.c

```diff
@@ -689,6 +689,9 @@ void dt_dev_load_image(dt_develop_t *dev, const uint32_t imgid)
   dt_dev_read_history(dev);
 
   dev->first_load = 0;
+
+  // Loading an image means we do some developing and so remove the darktable|problem|history-compress tag
+  dt_set_history_compress_problem(imgid, FALSE);
```

Comment:
Select corresponding tag in collection to see all images with problem. Go to darkroom, then back to lighttable the count of image is not updated and the image is still displayed eventhough the tag has been removed. I think we should update the collection here.

Reply:
But this is not easy, updating the collection means that the tag will be collapsed as I'm not sure we can update count on collect modules without reseting/redrawing the whole module. So maybe it is good as-is.

Reply:
I was not sure about this. Probably better to update the collection when closing darkroom?

Reply:
But then you're back into lighttable and the collection is fully collapsed. So I'm not sure.

Reply:
Ok, let me think about it

Reply:
What about this. If it hurts too much i could test and possibly remove tag and do the recalculation just in case.

But i think this is good anyway. We might alter more tags while developing...

Reply:
No this is not good to me. Select a tag in the collection module for example darktable|format|jpg go to darkroom back to lighttable and the "collect images" module tree is collapse. You need to reopen the tree... All context is lost!

Same if you select an entry in time, date.

Reply:
EDIT: no I was not testing the wrong code :( Sorry, indeed all is good now. Thanks.